buddyboss/buddyboss-platform
# Fixed - Default content &#8203; when adding gif and refactor gif module correctly

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1713 .

### How to test the changes in this Pull Request:

1. Go to activity post form
2. Click on add gif and any add gif and update the post without any content.
3. Check the activity content in db.

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### API PR: 
https://github.com/buddyboss/buddyboss-platform-api/pull/265
